spec/models/user_link_spec.rb

```diff
@@ -1,5 +1,22 @@
 require 'rails_helper'
 
 RSpec.describe UserLink, :type => :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let(:user){ create(:user) }
+
+  describe ".link_without_protocol" do
+    subject {UserLink.last.link_without_protocol}
+    context "when link has no protocol" do
+      before do
+        create(:user_link, link: 'foo.com')
```

Comment:
You just need one before with ```create(:user_link, link: link)```
Then you define link in each context with ```let(:link) { ... }```
You can then use ```subject{ link.link_without_protocol}``` or, better yet, ```subject{ link.without_protocol}```